src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -89,6 +91,8 @@ export class PendingRequestComponent implements OnInit, OnChanges {
         (reqs: PendingRequest[]) => {
           this.requests = reqs;
           this.status = reqs.length ? 'loaded' : 'empty';
+          if (this.status === 'loaded')
+            this.collapsed = false;
```

Comment:
As this status has been set at the previous line, it is clearer (especially in a subscribe) to base it on the "source", so just put `if (reqs.length)`.
```suggestion
          if (reqs.length) this.collapsed = false;
```